src/backend/models/letters.ts

```diff
@@ -66,7 +75,10 @@ export async function getAssignedLetters(userUuid: string): Promise<Array<Letter
          letters.*,
          users.email as assigned_responder_email,
          users.full_name as assigned_responder_full_name,
-         replies.status as reply_status
+         replies.status as reply_status,
+         replies.read_receipt,
+         replies.read_timestamp,
+         replies.status_timestamp
```

Comment:
I think these three lines need the ` as reply_read_receipt` to make them appear under correct names to fit the modified `LetterQueryResult`

I have not tested if this somehow works even now, I am so lazy that I will do that later 😄

Reply:
It's a valid problem